src/test/java/edu/hawaii/its/api/service/TestGrouperApiService.java

```diff
@@ -232,7 +232,7 @@ public void getSubjects() {
             assertTrue(testUhUuidsSet.contains(subject.getUhUuid()));
         }
 
-        // Numbers and Uids
+        // Uhuuids and Uids
         List<String> numbersAndUids = new ArrayList<>();
```

Comment:
Would be better to remove the comment then rename the `numbersAndUids` variable to `uhUuidsAndUids`.